docs/index.md

```diff
@@ -58,8 +58,8 @@ Contributing<reference/contributing>
 ```{toctree}
 :hidden:
 :caption: Extensions
-
 extensions/blender_addon
+extensions/unreal_engine
```

Comment:
Lets use a shorter label for the sidebar. perhaps,
`Unreal Engine<extensions/unreal_engine>`